static/audiofeed.yaml

```
@@ -1,3 +1,30 @@
+- title: 'Pixelplausch #04 - Über “intime” Videogames-Conventions und Entwicklungen bei AAA-Produktionen'
+  date: 2025-08-05T10:00
+  image: https://raw.githubusercontent.com/LucaNerlich/m10z/main/static/img/formate/cover/pixelplausch.jpg
+  description: |2-
+                  Mit der 4. Folge von Pixelplausch melden wir uns aus unserer Sommerpause zurück und sprechen mit André von Videospielgeschichten und unserem lieben M10Z-Kollegen Marcel aka kkuez über die folgenden Themen:
+
+                    - Diskussion über kleinere Videospiel-Messen als Alternative zur GamesCom.  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)